app/views/admin/users/show/reports/_index.html.erb

```diff
@@ -1,4 +1,4 @@
-<h2 class="crayons-subtitle-1">Reports submitted by @<%= @user.username %> (<%= @related_reports.count %>)</h2>
+<h2 class="crayons-subtitle-1">Reports submitted by <%= @user.name %> (<%= @related_reports.count %>)</h2>
 
 <% unless @related_reports.empty? %>
```

Comment:
Non-blocking: Ah, the dreaded `unless…else` clause.  Outside the scope, but I have a strong preference for avoiding the `unless...else` and flipping it to `if...else`.  The reasoning being that the else part of unless is basically `unless not` which is complicated.

But this is outside of the scope of the change, so is the musings of a curious reviewer.